cypress/fixtures/analysis.json

```diff
@@ -86,7 +78,7 @@
         "storyPoints": 1
     },
 
-    {
+    "Tackle-testapp-manuallyAnalyzePackages": {
```

Comment:
For this one too

Reply:
Here we provide ``` "manuallyAnalyzePackages": ``` and ``` "excludedPackagesList": ```

Reply:
Let's keep two tackle-testapp @kpunwatk .

1) tackle-testapp (with no advance options)
2) tackle-testapp-with-advanced-options(This will have all other options like excludepackage, excluderule, enabletagging etc)